presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -578,6 +578,33 @@ The default value of 60 gb is calculated based on available machine memory of 64
 Specifies the amount of memory to shrink when the memory pushback is
 triggered. This only applies if ``system-mem-pushback-enabled`` is ``true``.
 
+``system-mem-pushback-abort-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+If true, memory pushback will abort queries with the largest memory usage under
+low memory condition. This only applies if ``system-mem-pushback-enabled`` is ``true``.
+
+``worker-overloaded-threshold-mem-gb``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
```

Comment:
It seems a bit odd to use 0 for default threshold. Can we use something like -1 to indicate that it can be ignored ?

Reply:
@aditi-pandit 
Thanks for reviewing this PR, seem like hard to find anyone brave enough these days to review a small change.

Using zeros for 'no op' is the practice we use already in the config.
See lines:
252 for kDriverCancelTasksWithStuckOperatorsThresholdMs
296 for kSystemMemLimitGb
365 for kAsyncCachePersistenceInterval
451 for kSharedArbitratorMaxMemoryArbitrationTime
532 for kSharedArbitratorGlobalArbitrationMemoryReclaimPct

It actually makes a certain semantic sense - threshold 0 for something that cannot be negative does not make sense, so it can be used as the 'magic number'.